mmv1/templates/terraform/examples/privateca_quickstart.tf.erb

```diff
@@ -0,0 +1,97 @@
+# [START privateca_quickstart]
+provider google{}
+provider tls{}
```

Comment:
Sorry - we're not able to run tests with non-google providers, usually.  Do you think there's a way to do this without the TLS provider?  Otherwise this will need to get disabled for testing, or we'll need to add the TLS provider to our tests (unlikely).

Reply:
We will need to retain the tls provider to create a CSR that's exportable. 
Is disabling a better option in this scenario?

Reply:
Yep, you won't be able to run this as a test.